templates/profile/prometheus/exporter/node/rubyHealthCheck-testruby.sh.erb

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+VER="$1"
+if [ -z "$VER" ]; then
+  echo 'Usage ./testruby.sh <version>'
+  exit 1
+fi
+mkdir "$VER"
+pushd "$VER"
+echo "$VER" > .ruby-version
+curl ae-files/ruby-check.tar | tar -x --strip-components=1
+./run.sh
```

Comment:
These relative paths were convenient in piecing things together, but they could be brittle. I guess they already assume that they are being run in the script dir, so it's fine -- just noting that this is less robust than using dirname for script-relative or absolute.

I think the most robust answer here would be to have the cron job run in a clean temp directory and call the script from there. In that case, resolving the path to the running script is probably necessary. As always, it's more complicated than you would hope, but there is a copypasta recipe: https://stackoverflow.com/a/246128

Reply:
I may not be understanding this one completely.  I get that absolute paths are better.  However, when adding that copypasta (assuming i have it in the right place), it tries to run ./run.sh from outside the version directory.  I'm not entirely sure where to put it.    

Perhaps just doing "/usr/loca/rubytests/$VER/run.sh" instead of "./run.sh" in this case is acceptable?

Reply:
Well, the cron job will invoke the testall script, but should probably cd to a temp dir. From there, we know where testruby will be, so we should invoke that absolutely. It naturally cd's into the specific version, so the relative ./run.sh there is no problem.